### ISSUE TITLE
Still won't open at all on PikaOS 4 (Debian/Linux)

### ISSUE THREAD

**Issue by Jakeh9777** [vrcx-team/VRCX#1001]
I ran the script and it installed. Looks like its running but nothing shows up at all like I mentioned above when I installed it through Lutris. 
![image](https://github.com/user-attachments/assets/9000e3c3-65ba-46b8-b5ba-aca3b82e7984)

_Originally posted by @Jakeh9777 in https://github.com/vrcx-team/VRCX/issues/938#issuecomment-2515193345_

**Comment by Natsumi-sama** [comment]
Try run it with the provided shortcut/runscript created by the install script.

**Comment by Natsumi-sama** [comment]
Script has since been updated once again.

**Comment by Jakeh9777** [comment]
Nope, I'm guessing it does not work on Gnome 47 or Debian. Because we got it to work on Nobara which is fedora and it was running with gnome 46 instead.

**Comment by Jakeh9777** [comment]
I was having issues with the OS on some other things so  I reinstalled my OS with a clean install. Now I get this when running the script: 
![image](https://github.com/user-attachments/assets/b1bb3be3-0014-4745-a735-fbdaff1e26fb)

**Comment by Jakeh9777** [comment]
Any updates?

**Comment by rs189** [comment]
> Any updates?

There is a native Linux version now please try that instead of Wine.

**Comment by Jakeh9777** [comment]
> > Any updates?
> 
> There is a native Linux version now please try that instead of Wine.

Oh there is?! I'll check after work tonight. Thank you so much.

**Comment by Jakeh9777** [comment]
I'm not seeing the appimage anywhere.

**Comment by Natsumi-sama** [comment]
Only nightly is currently available, stable is yet to be released.
